tutorials/assets_pipeline/importing_images.rst

```diff
@@ -381,7 +381,7 @@ Roughness > Mode
 The color channel to consider as a roughness map in this texture. Only effective if
 **Roughness > Src Normal** is not empty.
 
-Rougness > Src Normal
+Roughness > Src Normal
 ^^^^^^^^^^^^^^^^^^^^^
```

Comment:
```suggestion
^^^^^^^^^^^^^^^^^^^^^^
```

Reply:
Should be fixed, I hope I did that right?